libs/build_api_v2.py

```diff
@@ -141,7 +180,7 @@ def build_annotations(one_region: OneRegionTimeseriesDataset, log) -> Annotation
         annotations_name: _build_metric_annotations(one_region, field_name, log)
         for annotations_name, field_name in name_and_common_field
     }
-    return Annotations(**annotations)
+    return {k: v for k, v in annotations.items() if v is not None}
```

Comment:
do you plan to merge with a dict or `Annotations`?

Reply:
Ah i should revert this